Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md

```diff
@@ -11,7 +20,7 @@ There are two application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow us access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).
+To allow us to access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).
```

Comment:
Who is "us"/"our"?

Reply:
i didnt write this line, was there before. 
I think "us" refers to Cortex Xsoar, I see theres similiar phrasing in other integrations that need an app added (especially with microsoft)

Reply:
i'll switch this here